demo/src/app/default/default.component.html

```diff
@@ -2,8 +2,9 @@
   <div class="container">
     <img class="bd-booticon" src="img/logo.svg"/>
     <p class="lead hidden-md-down">
-      Bootstrap 4 components, powered by Angular and written by the angular-ui Team.
+      Bootstrap 4 components, powered by Angular
     </p>
+    <p>Currently at v{{version}}</p>
```

Comment:
Yeh, maybe. But TBH I'm not sure how to do it since we don't have any obvious mapping between version no and GH tags in the changelog (those will look like https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CHANGELOG.md#100-alpha23-2017-04-12) <- notice the date

Given that this PR doesn't change this part (just moves it around) and the technical difficulty involved I would rather tackle this in a separate PR / issue.